Package.swift

```diff
@@ -15,5 +15,5 @@ let package = Package(
         sources: ["Sources"]
       )
     ],
-    swiftLanguageVersions: [3, 4, 5]
+    swiftLanguageVersions: [.v4, .v4_2, .v5]
```

Comment:
That's not the only problem of SPM setup :)
Even with this fixes you won't be able to build this package. Please check other PRs :)